.github/workflows-source/ci-main.yml

```diff
@@ -4,14 +4,13 @@ on:
   merge_group:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
-  schedule: # Todo: switch to push on master and remove schedule once we have more macos runners
-    - cron: "0 * * * *"
-  pull_request:
-  # Used as reusable workflow within release-testing workflow
-  workflow_call:
   push:
     branches:
+      - master
       - 'dev-gh-*'
```

Comment:
Does anyone know what `dev-gh-*` is used for and if it's still needed?

Reply:
It's for development purposes if anyone wants to do develop only on branch.

Reply:
> @marko-k0 can you add a note about this in the workflow file?

Sure.